pkg/controller/manager/manager_controller.go

```diff
@@ -167,8 +167,10 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 		return fmt.Errorf("manager-controller failed to watch the '%s' namespace: %w", common.TigeraPrometheusNamespace, err)
 	}
 
-	if err = utils.AddConfigMapWatch(managerController, render.ECKLicenseConfigMapName, render.ECKOperatorNamespace, eventHandler); err != nil {
-		return fmt.Errorf("manager-controller failed to watch the ConfigMap resource: %v", err)
+	if !opts.ElasticExternal {
```

Comment:
I wonder if this watch is actually needed at all? I don't see the controller ever reference this configmap.

Reply:
:shrug: